cedar-policy/CHANGELOG.md

```diff
@@ -43,6 +43,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - For the `partial-eval` experimental feature: make the return values of
   `RequestBuilder`'s `principal`, `action`, `resource`, `context` and
   `schema` functions `#[must_use]`. (#502)
+- For the `partial-eval` experimental feature: make `RequestBuilder::schema`
+returns a `ValidatedRequestBuilder` so that `ValidatedRequestBuilder::build`
+method checks the request against the schema provided and `RequestBuilder::build` 
```

Comment:
```suggestion
return a `ValidatedRequestBuilder` so that the `ValidatedRequestBuilder::build`
method checks the request against the schema provided and the `RequestBuilder::build` 
```